elasticsearch/dsl/document_base.py

```diff
@@ -602,3 +603,44 @@ def to_dict(self, include_meta: bool = False, skip_empty: bool = True) -> Dict[s
 
         meta["_source"] = d
         return meta
+
+    @classmethod
+    def _get_field_names(
+        cls, for_esql: bool = False, nested_class: Optional[type[InnerDoc]] = None
```

Comment:
Should this be named `ignore_missing_fields` too? After all, ES|QL may support nested objects at some point.